angio-frontend/src/modules/analyse/components/AnalyseListFilter.vue

```diff
@@ -0,0 +1,149 @@
+<template xmlns:v-slot="http://www.w3.org/1999/XSL/Transform">
+    <v-card flat>
+        <v-card-title>
+            <v-layout row wrap>
+                <v-flex xs12>
+                    <v-layout align-end>
+                        <v-flex xs7>
+                            <v-text-field
+                                    v-model="search"
+                                    v-bind:label="$t('analyse.component.analyseListFilter.search')"
+                            ></v-text-field>
+                        </v-flex>
+
+                        <v-flex xs5 pl-2>
+                            <v-combobox
+                                    v-model="selectedStatuses"
+                                    v-bind:items="statuses"
+                                    v-bind:item-text="(item) => $t(item.text)"
+                                    v-bind:label="$t('analyse.component.analyseListFilter.statusField')"
+                                    multiple
+                                    clearable
+                                    small-chips
+                            >
+                                <template v-slot:selection="data">
+                                    <v-chip
+                                            v-bind:key="JSON.stringify(data.item)"
+                                            v-bind:selected="data.selected"
+                                            v-bind:disabled="data.disabled"
+                                            class="v-chip--select-multi caption"
+                                            v-on:input="data.parent.selectItem(data.item)"
+                                            small
+                                    >
+                                        {{ $t(data.item.text) }}
+                                        <v-icon
+                                                small
+                                                v-on:click="data.parent.selectItem(data.item)"
```

Comment:
Attributes with prefix 'v-' must be always on top

Reply:
Doce